component/FloatingAction.js

```diff
@@ -396,6 +396,18 @@ class FloatingAction extends Component {
     }
     this.reset();
   }
+  
+  _updateState = (nextState, callback) => {
```

Comment:
we don't use underscore for private functions, just rename it for `updateState` please :)